src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -35,6 +35,11 @@ public abstract class AppiumDriver : WebDriver,
     {
         private const string NativeApp = "NATIVE_APP";
 
+        /// <summary>
+        /// The default command timeout for HTTP requests in a AppiumDriver instance.
+        /// </summary>
+        private static new readonly TimeSpan DefaultCommandTimeout = TimeSpan.FromSeconds(600.0);
```

Comment:
where this value is used?

Reply:
https://github.com/appium/dotnet-client/blob/30d9dec6f4966b9bbd8ac687a2576efda8c0ac1c/src/Appium.Net/Appium/AppiumDriver.cs#L73

https://github.com/appium/dotnet-client/blob/30d9dec6f4966b9bbd8ac687a2576efda8c0ac1c/src/Appium.Net/Appium/AppiumDriver.cs#L78

https://github.com/appium/dotnet-client/blob/30d9dec6f4966b9bbd8ac687a2576efda8c0ac1c/src/Appium.Net/Appium/AppiumDriver.cs#L94

https://github.com/appium/dotnet-client/blob/30d9dec6f4966b9bbd8ac687a2576efda8c0ac1c/src/Appium.Net/Appium/AppiumDriver.cs#L99

Reply:
Do I understand it right, - is it the same property defined protected/public in the parent class?

Reply:
Correct

Reply:
I am not an expert in dotnet, but in other languages it is usually not a good practice "reduce" member visibility level in descendants. That is why it makes sense to make this field either protected or public if it was defined as protected in the parent class.
Or simply rename it if you prefer to keep that private.

Reply:
Changed to protected